extensions/wikia/FacebookPreferences/modules/ext.wikia.facebookPreferences.css

```diff
@@ -0,0 +1,9 @@
+.wds-button.facebook-button, .wds-button.facebook-button:hover {
+    background: #4b6ea8;
```

Comment:
Why this color btw?
Wouldn't we prefer to use FB color or another variable from WDS?
http://fandomdesignsystem.com/#/base-styles/colors